packages/solid-crs-manage/lib/public/nl-NL.json

```
@@ -203,6 +204,17 @@
                         "title": "Eenheid"
                     }
                 }
+            },
+            "loan": {
+                "title": "Bruikleen",
+                "subtitle": {
+                    "owner": "Uitgeleend object.",
+                    "loanee": "Dit object wordt uitgeleend door <a href=\"{{href}}\" target=\"_blank\"> {{institution}} </a>"
```

```suggestion
                    "lender": "Dit object wordt uitgeleend door <a href=\"{{href}}\" target=\"_blank\"> {{institution}} </a>"
```

from https://github.com/netwerk-digitaal-erfgoed/solid-crs/discussions/557